class/airlock-microgateway.yml

```
@@ -1,11 +1,94 @@
+      'False':
+        input_paths: []
         input_type: jsonnet
-        output_path: airlock-microgateway/
+        output_path: ''
+    olm:
+      dependencies:
+        - type: https
+          source: ${airlock_microgateway:gateway_api:source}
+          output_path: ${_base_directory}/dependencies/gateway-api/standard-install.yaml
+      compile:
+        - input_paths:
+            - ${_base_directory}/component/app.jsonnet
+          input_type: jsonnet
+          output_path: .
+
+        - input_paths:
+            - ${_base_directory}/component/namespace.jsonnet
+          input_type: jsonnet
+          output_path: airlock-microgateway/00_prerequisites/
+
+        - input_paths:
+            - ${_base_directory}/component/olm.jsonnet
+          input_type: jsonnet
+          output_path: airlock-microgateway/01_olm/
+          # If multi-tenant:
+          # output_path: ${_instance}/olm/
+        - ${_kapitan:k8s_gw_api:${airlock_microgateway:gateway_api:enabled}}
+        - input_paths:
+            - ${_base_directory}/component/main.jsonnet
+          input_type: jsonnet
+          output_path: airlock-microgateway/00_prerequisites/
+
+    helm:
+      dependencies:
+        - type: https
+          source: ${airlock_microgateway:gateway_api:source}
+          output_path: ${_base_directory}/dependencies/gateway-api/standard-install.yaml
+        - type: helm
+          chart_name: ${_helm_chart_name}
+          version: ${airlock_microgateway:charts:${_helm_chart_name}:version}
+          source: ${airlock_microgateway:charts:${_helm_chart_name}:source}
+          output_path: ${_base_directory}/helmcharts/airlock-microgateway/${airlock_microgateway:charts:${_helm_chart_name}:version}/
+
+      compile:
+        - input_paths:
+            - ${_base_directory}/component/app.jsonnet
+          input_type: jsonnet
+          output_path: .
+
+        # - input_paths: ${_kapitan:jsonnet_input_paths}
+        #   input_type: jsonnet
+        #   output_path: ${_instance}/
+
+        - input_paths:
+            - ${_base_directory}/component/namespace.jsonnet
+          input_type: jsonnet
+          output_path: airlock-microgateway/00_prerequisites/
+        - ${_kapitan:k8s_gw_api:${airlock_microgateway:gateway_api:enabled}}
+        - input_paths:
+            - ${_base_directory}/component/render-helm-values.jsonnet
+          input_type: jsonnet
+          output_path: ${_base_directory}/helm_values/
+        - output_path: airlock-microgateway/01_airlock-microgateway_helmchart/
+          input_type: helm
+          output_type: yaml
+          input_paths:
+            - ${_base_directory}/helmcharts/airlock-microgateway/${airlock_microgateway:charts:${_helm_chart_name}:version}/
+          helm_params:
+            name: ${_release_name}
+            namespace: ${airlock_microgateway:namespace}
+          helm_values_files:
+            - ${_base_directory}/helm_values/values.yaml
```

Replace this with `helm_values: ${airlock_microgateway:helm_values}` since we don't render anything.

```suggestion
          helm_values: ${airlock_microgateway:helm_values}
```